pkg/ddl/ddl_api.go

```diff
@@ -96,6 +96,8 @@ const (
 	tiflashCheckPendingTablesRetry = 7
 )
 
+var errCheckConstraintIsOff = errors.NewNoStackError("tidb_enable_check_constraint is off")
```

Comment:
s/`tidb_enable_check_constraint is off`/`variable.TiDBEnableCheckConstraint+"is off"` ?